include/seqan3/alphabet/adaptation/char.hpp

```diff
@@ -69,10 +69,12 @@ namespace seqan3::detail
 //!\brief The list of types that are defined as char adaptations.
 using char_adaptations            = meta::list<char,
                                                char16_t,
-                                               char32_t>;
+                                               char32_t,
+                                               wchar_t>; // TODO Windows adaption requires different handling of wchar_t because in Windows it is 16 bits and holds UTF-16 code units
```

Comment:
This line is longer than 120 characters.

(See https://github.com/seqan/seqan3/wiki/General-formatting )